include/rvvlm_lgammaD.inc.h

```diff
@@ -480,7 +480,7 @@
   } while (0)
 
 void F_VER1(API) {
-  size_t vlen;
+  size_t vlen = VSETMAX();
```

Comment:
It should be initialized to `VSET(_inarg_n);`, so it matches https://github.com/rivosinc/veclibm/pull/32/commits/0aff5807125da1fee7c79b45612d46d71f78dbf6#diff-72fcde0d45991c232db8b3be07c8c9fab9a8d09c48f6d3786aea5411074f3eabR491